tests/integer_list_test.py

```diff
@@ -0,0 +1,81 @@
+import girder_worker
+import unittest
+
+
+class TestIntegerList(unittest.TestCase):
+
+    def setUp(self):
+        self.analysis = {
+            "name": "concatenate",
+            "inputs": [
+                {
+                    "name": "a",
+                    "type": "integer_list",
+                    "format": "integer_list",
+                    "default": {
+                        "format": "json",
+                        "data": "[0]"
+                    }
+                },
+                {
+                    "name": "b",
+                    "type": "integer_list",
+                    "format": "integer_list"
+                }
+            ],
+            "outputs": [{"name": "c",
+                         "type": "integer_list", "format": "integer_list"}],
+            "script": "c = a + b",
+            "mode": "python"
+        }
+
+    def test_integer(self):
```

Comment:
I just followed the naming convention used for the [rest of the tests](https://github.com/girder/girder_worker/tree/master/tests) in girder_worker.